cfme/web_ui/menu.py

```diff
@@ -121,8 +142,8 @@ def get_rid_of_the_menu_box():
     ),
     ('infrastructure', 'Infrastructure'): (
```

Comment:
@seandst I'd like to have you opinion on this file's changes. I introduced callables so we can decide what string to use at runtime.